docker-compose.yml

```diff
@@ -21,6 +21,7 @@ services:
     volumes:
       - wallet-${NETWORK}-db:/wallet-db
       - node-ipc:/ipc
+      - ./config:/config
```

Comment:
I believe this change makes cardano-wallet get configs from local machine that are based in local `./config` directory. cardano-wallet docker image has configs inside so this should not be necessary. However, it seems that docker-compose does not properly start on `preview` after recent `preview` network re-spin. Added bug ticket for that: https://input-output.atlassian.net/browse/ADP-2359.